cpp/benchmarks/common/generate_input.cu

```diff
@@ -250,17 +311,53 @@ auto create_run_length_dist(cudf::size_type avg_run_len)
   return std::gamma_distribution<float>{alpha, avg_run_len / alpha};
 }
 
-// identity mapping, except for bools
-template <typename T, typename Enable = void>
-struct stored_as {
-  using type = T;
-};
-
-// Use `int8_t` for bools because that's how they're stored in columns
-template <typename T>
-struct stored_as<T, std::enable_if_t<std::is_same_v<T, bool>>> {
-  using type = int8_t;
-};
+/**
+ * @brief Generate indices within range [0 , cardinality) repeating with average run length
+ * `avg_run_len`
+ *
+ * @param avg_run_len  Average run length of the generated indices
+ * @param cardinality  Range of the generated indices
```

Comment:
I would use `max` or `upper_bound` instead. `cardinality` implies other things to me.

Reply:
Upon a complete review, I realize that this change would require modifying a ton of extra code. I think this change can be postponed or skipped altogether.

Reply:
Cardinality should be the right term here. We are generating `num_rows` values, with at most `cardinality` unique values. AFAIK, that's what cardinality is. Maybe the description in the comment is misleading.